src/components/Registers.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Category from './Category';
+
+const RegisterPanel = ({ registers }) => {
+  const rs = registers.map((reg, i) => (
+    <li key={`{register-${i.toString()}}`}>
+      x
+      { i }
+      :
+      { reg.low_}
+    </li>
+  ));
+  return (
+    <div>
```

Comment:
is the div the right element for this? is there something that would make more sense semantically? should we add an `id` to this container element?

Reply:
im not too sure which html tag would be best yet, do you have any thoughts? Im thinking this would come to fruition when we start styling the components

Reply:
a `div` may be the right choice. i just want to make sure we're thinking about the HTML we're choosing